Library/Formula/libvisio.rb

```diff
@@ -15,10 +14,11 @@ class Libvisio < Formula
   depends_on "pkg-config" => :build
   depends_on "boost" => :build
   depends_on "cppunit" => :build
-  depends_on "libwpd"
-  depends_on "libwpg"
-  depends_on "icu4c"
   depends_on "librevenge"
+  depends_on "icu4c"
+  depends_on "libxml2"
+  depends_on "homebrew/dupes/gperf"
+  depends_on "homebrew/dupes/zlib"
```

Comment:
They haven't been committed here. Squash commits by running `brew update; git checkout $YOUR_BRANCH; git rebase --interactive origin/master` and then mark each commit other than the first as `squash` or `fixup` then `git push -f`. Thanks!